src/parkapi_sources/converters/heidelberg/models.py

```diff
@@ -86,40 +86,62 @@ class HeidelbergParkingSubType(Enum):
 
 @validataclass
 class HeidelbergInput:
-    acceptedPaymentMethod: HeidelbergPaymentMethodType = RemoveValueDict(
-        ListValidator(EnumValidator(HeidelbergPaymentMethodType)),
+    id: str = StringValidator()
+    acceptedPaymentMethod: Optional[HeidelbergPaymentMethodType] = (
+        NoneableRemoveValueDict(
+            ListValidator(EnumValidator(HeidelbergPaymentMethodType)),
+        ),
+        Default(None),
     )
-    addressLocality: str = RemoveValueDict(StringValidator())
-    availableSpotNumber: Optional[int] = NoneableRemoveValueDict(IntegerValidator()), Default(None)
-    closingHours: time = RemoveValueDict(TimeValidator(time_format=TimeFormat.NO_SECONDS))
-    description: str = RemoveValueDict(ReplacingStringValidator(mapping={'\r': '', '\n': ' ', '\xa0': ' '}))
-    facilities: list[str] = RemoveValueDict(ListValidator(EnumValidator(HeidelbergFacilityType)))
-    familyParkingSpots: int = RemoveValueDict(IntegerValidator())
-    googlePlaceId: str = RemoveValueDict(StringValidator())
-    handicappedParkingSpots: int = RemoveValueDict(IntegerValidator())
-    images: list[str] = RemoveValueDict(ListValidator(UrlValidator()))
+    addressLocality: Optional[str] = NoneableRemoveValueDict(StringValidator()), Default(None)
+    availableSpotNumber: Optional[int] = NoneableRemoveValueDict(IntegerValidator(min_value=0)), Default(None)
+    closingHours: Optional[time] = (
+        NoneableRemoveValueDict(TimeValidator(time_format=TimeFormat.NO_SECONDS)),
```

Comment:
Any cases where you actually need the `NoneableRemoveValueDict`, and not `Noneable`, or even nothing of it, because if it's set, it's never None? Don't see them in test data, but maybe I miss something?